apps/website/src/examples/Badge.soft.tsx

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Badge } from '@itwin/itwinui-react';
+
+export default () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        columnGap: 4,
+        placeItems: 'center',
+        flexWrap: 'wrap',
+        justifyContent: 'center',
```

Comment:
Should we add some margin/padding? Because the badges stick to the edges in some screen widths.

![image](https://user-images.githubusercontent.com/45748283/213263305-5c1a3e51-dc10-43d0-8c53-dae50ad55e09.png)


<!-- ![image](https://user-images.githubusercontent.com/45748283/213263159-306f70fb-99f9-4cd8-b201-a07e5bc2bf04.png) -->

```suggestion
        justifyContent: 'center',
        margin: '10px',
```